templates/al2023/template.json

```diff
@@ -295,6 +295,12 @@
       "inline": [
         "sudo rm -rf {{user `working_dir`}}"
       ]
+    },
```

Comment:
Any reason we can add this to install-worker.sh after downloading kubelet? Seems odd for it to be a separate provisioner